### PR TITLE
Add KRPullLoader

### PR DESCRIPTION
**Pull request by krimpedance**

### CODE REVIEW

File: Animation/pull-to-refresh.md [hunk 1]
```diff
@@ -58,8 +58,8 @@
 
 ![](https://camo.githubusercontent.com/dbdf82cdf90a8ded67c129b91a558e25aac08eb2/68747470733a2f2f6431337961637572716a676172612e636c6f756466726f6e742e6e65742f75736572732f32313235382f73637265656e73686f74732f323032323836322f6174746163686d656e74732f3335373932302f616e696d6174696f6e2e676966)
 
-[PullToBounce](https://github.com/entotsu/PullToBounce)
+[KRPullLoader](https://github.com/krimpedance/KRPullLoader)
```
Comment (onmyway133) [inline Animation/pull-to-refresh.md]: Hi, you shouldn't replace other people 's stuff. Mind fixing this ?

Comment (krimpedance) [reply Animation/pull-to-refresh.md]: That's my mistake.
I'll correct immediately.

**Comment by krimpedance** [comment]
@onmyway133 
I corrected.
Please merge.
Thank you.

**Review by hemangshah — APPROVED** [review]

**Comment by onmyway133** [comment]
@krimpedance @hemangshah Hi, this is a bit simple, so I will close it for now. Hope you understand

**Pull request closed without merging**